calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detections/board_detection.py

```diff
@@ -179,11 +165,8 @@ def get_pose(self, model: Optional[CameraModel] = None) -> Tuple[np.array, np.ar
 
         return self._cached_pose
 
-    def get_flattened_3d_points(self, model: Optional[CameraModel] = None) -> np.array:
+    def get_flattened_3d_points(self, model: CameraModel) -> np.array:
```

Comment:
The following lines are a bug from before this PR, but can you fix them here?

```
if model == self._cached_camera_model and self._cached_flattened_3d_points is not None:
            return self._cached_pose
```

Reply:
Addressed in https://github.com/tier4/CalibrationTools/pull/208/commits/d966e0376241ef319b1b72fc814da0f36be0eeba